src/commands/sources/query.ts

```diff
@@ -81,7 +82,8 @@ export class QuerySourcesCommand extends Command {
       level: flags.loglevel,
     });
     const catalog = await Catalog.default();
-    const service = new DistributionsService({logger, catalog});
+    const comunica = newEngine();
```

Comment:
Looks good! We may need to change this at some point, though: this Comunica instance is bound to SPARQL endpoints. If we want to query other source types (e.g. custom Web APIs) we'll have to use different Comunica instances.